packages/lab/src/badge/Badge.css

```
@@ -24,44 +6,35 @@
   position: relative;
   flex-shrink: 0;
   vertical-align: middle;
-  line-height: var(--salt-text-lineHeight);
-  font-size: var(--salt-text-fontSize);
-}
-
-/* Give automatic margin when they sit next to each other */
-.saltBadge + .saltBadge {
-  margin-left: var(--badge-sequence-gap);
-}
 
-/* Style applied to the badge in the corner of the node */
-.saltBadge-badge {
-  color: var(--badge-content-text-color);
-  display: inline-block;
-  font-weight: 700; /* TODO: Use characteristic once aligned */
-  font-size: var(--badge-fontSize);
-  /* default small type */
-  width: auto;
-  min-width: 15px;
-  line-height: var(--badge-content-lineHeight);
-  border-radius: var(--badge-content-borderRadius);
-  text-align: center;
+  /* Should this vary according to touch size */
+  padding: 0px var(--salt-spacing-50);
+  line-height: var(--salt-text-lineHeight);
+  min-width: 10px;
+  line-height: 15px;
+  border-radius: 16px;
```

If we want a full border radius no matter the height of the badge, this should be 
```suggestion
  border-radius: 9999px;
```